fastlane/lib/fastlane/actions/delete_keychain.rb

```diff
@@ -22,13 +33,20 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :name,
                                        env_name: "KEYCHAIN_NAME",
                                        description: "Keychain name",
-                                       optional: false)
+                                       conflicting_options: [:path],
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :path,
```

Comment:
We can't change option names, as this would break setups, let's leave it like it is for now 👍